tests/run-make/raw-dylib-link-ordinal/Makefile

```diff
@@ -1,17 +0,0 @@
-# Test the behavior of #[link(.., kind = "raw-dylib")] and #[link_ordinal] on windows-msvc
```

Comment:
Remark: this comment is inaccurate since it's windows, not just windows-msvc.

Reply:
Indeed, looks like it just got mass copy pasted when these tests were created. It's not the only inaccurate one.